src/components/MultiSelect/MultiSelect.js

```diff
@@ -251,7 +267,16 @@ export default class MultiSelect extends React.Component {
                 {isOpen && (
                   <ListBox.Menu>
                     {componentsX
```

Comment:
@emyarod Are `v9`/`v10` code here identical besides sorting? Of so, should the code branch here be based on the "sorting mode"?

Reply:
I removed the branching, because the 2 branches are completely identical after extracting the sorting

Reply:
yes @asudoh the code is identical besides sorting at this point. https://github.com/IBM/carbon-components-react/issues/2109 will introduce some v10 only changes but it is still WIP (https://github.com/emyarod/carbon-components-react/blob/2109-match-multiselect-to-experimental-spec/src/components/MultiSelect/MultiSelect.js) and shouldn't conflict with the sorting behavior we are adding in this PR